konduit-serving-cli/src/main/java/ai/konduit/serving/cli/launcher/command/build/extension/ProfileCommand.java

```diff
@@ -0,0 +1,509 @@
+    @DefaultValue("ch.qos.logback:logback-classic:1.2.3")
+    @Description("One or more space separated values (maven coordinates) indicating additional dependencies to be included with " +
+            "the server launch. The pattern of additional dependencies should be either <group_id>:<artifact_id>:<version> or " +
+            "<group_id>:<artifact_id>:<version>:<classifier>")
+    public void setAdditionalDependencies(List<String> additionalDependencies) {
+        this.additionalDependencies = additionalDependencies;
+    }
+
+    private enum SubCommand {
+        DEFAULT, CREATE, LIST, VIEW, EDIT, DELETE
+    }
+
+    @Override
+    public void run() {
+        if(profileName == null && !this.subCommand.equals(SubCommand.LIST)) {
+            out.println("Please specify a profile name.");
+            System.exit(1);
+        }
+
+        switch (this.subCommand) {
+            case DEFAULT:
+                setDefaultProfile(profileName);
+                break;
+            case LIST:
+                listProfiles();
+                break;
+            case EDIT:
+                editProfile();
+                break;
+            case DELETE:
+                deleteProfile(profileName);
+                break;
+            case VIEW:
+                viewProfile(profileName);
+                break;
+            case CREATE:
+                createProfile();
+                break;
+        }
+    }
+
+    public static void setDefaultProfile(String profileName) {
+        if(!isProfileExists(profileName)) {
+            log.error("No profile with name {} exists.", profileName);
+        } else {
+            try {
+                FileUtils.writeStringToFile(defaultProfileNamePath, profileName, StandardCharsets.UTF_8);
+                log.error("Successfully set '{}' profile as default.", profileName);
+            } catch (IOException e) {
+                log.error("Unable to set default profile", e);
+            }
+        }
+    }
+
+    public static Profile getDefaultProfile() {
+        try {
+            String profileName = FileUtils.readFileToString(defaultProfileNamePath, StandardCharsets.UTF_8);
+            return getProfile(profileName);
+        } catch (IOException e) {
+            log.error("Unable to get default profile", e);
+            return null;
+        }
+    }
+
+    private void createProfile() {
+        if(isProfileExists(profileName)) {
+            out.format("Profile with name %s already exists.%n", profileName);
+        } else {
+            saveProfile(profileName, fillProfileValues(new Profile()));
+        }
+    }
+
+    private void editProfile() {
+        if(isProfileExists(profileName)) {
+            if(profileName.equals("CPU") || profileName.equals("CUDA")) {
+                out.format("Cannot edit pre-set profiles with name 'CPU' or 'CUDA'.%n");
+                System.exit(1);
```

Comment:
We should move these contants ("CPU", "CUDA", "x86" etc) to `static final String` fields on the class.

Also, we should probably allow editing of these CPU and CUDA profiles, for two reasons:
(a) If the user can't run AVX2 for example and wants to default to x86 only, or
(b) If the user installs a different CUDA version after KS installation